CONTRIBUTING.md

```diff
@@ -165,3 +165,27 @@ which tests have failed and then fix them by making more changes. Just use
 `git push pullrequest` to publish your changes. The CI tests will
 run with your updated code. Use `hub ci-status --verbose` to monitor
 their status.
+
+## Flags
+
+V allows you to pass custom flags using `-d my_flag` that can then be checked at compile time (see the documentation about [compile-time if](https://github.com/vlang/v/blob/master/doc/docs.md#compile-time-if))
+There are numerous flags that can be passed to the compiler to help you when debugging.
+This is an exhaustive list of all flags that can be passed down to the compiler, and what they do.
+
+| Flag | Usage |
+|------|-------|
+| `debugautostr` | Prints informations about `.str()` method auto-generated by the compiler during C generation |
+| `debugscanner` | Prints debug information during the scanning phase |
+| `debug_codegen` | Prints automatically generated V code during the scanning phase |
+| `debug_interface_table` | Prints generated interfaces during C generation |
+| `debug_interface_type_implements` | Prints debug information when checking that a type implements in interface |
+| `noskip` | Forces not to skip any file when testing the compiler or vlib |
```

Comment:
`noskip` is not passed to the V compiler itself, it is for `vlib/v/compiler_errors_test.v` and `vlib/v/tests/valgrind/valgrind_test.v` . I think it should not be in this table.